features/com.palladiosimulator.textual.repository.feature/feature.xml

```diff
@@ -1,23 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<feature id="com.palladiosimulator.textual.repository.feature"
-	label="RepoLang Feature "
-	version="1.0.0.qualifier">
-	<plugin
-			id="com.palladiosimulator.textual.repository"
-			download-size="0"
-			install-size="0"
-			version="0.0.0"
-			unpack="false"/>
-	<plugin
-			id="com.palladiosimulator.textual.repository.ide"
-			download-size="0"
-			install-size="0"
-			version="0.0.0"
-			unpack="false"/>
-	<plugin
-			id="com.palladiosimulator.textual.repository.ui"
-			download-size="0"
-			install-size="0"
-			version="0.0.0"
-			unpack="false"/>
+<feature
```

Comment:
Optional: integration of Palladio license feature

Reply:
Done